manifests/c/CreativeTechnologyLtd/SoundBlasterCommand/3.4.92.00/CreativeTechnologyLtd.SoundBlasterCommand.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate
+
+PackageIdentifier: CreativeTechnologyLtd.SoundBlasterCommand
+PackageVersion: 3.4.92.00
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://files.creative.com/manualdn/Applications/100330/dccAqktyLO/AECMDMasterInstaller_3.4.92.00.exe
+  InstallerSha256: DE40104936219BD1AE32304405E1144F07CD6B81CD1D219CF2BACC7991B70BE3
+ManifestType: installer
+InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent

```

Comment:
```suggestion
InstallerSwitches:
    Custom: /NORESTART
```

Reply:
You don't need to add Silent switches when the installer type is inno/nullsoft, also this switch shoould work if i typed it correctly for inno